exporter/common.go

```diff
@@ -63,7 +66,15 @@ func listCollections(ctx context.Context, client *mongo.Client, database string,
 		return nil, errors.Wrap(err, "cannot get the list of collections for discovery")
 	}
 
-	return collections, nil
+	filteredCollections := []string{}
+	for _, collection := range collections {
+		if collection == systemCollections[0] {
```

Comment:
if there is only one element in the systemCollections slice, could we use it as a constant variable?

Or if you plan to add more elements to `systemCollections` slice later could you please write this code which will work if the slice has two or more elements, but not only one?